static/js/modules/columns.js

```diff
@@ -68,34 +236,111 @@ var filings = {
     },
   },
   amendment_indicator: amendmentIndicatorColumn,
-  receipt_date: tables.dateColumn({data: 'receipt_date', className: 'min-tablet'}),
-  coverage_end_date: tables.dateColumn({data: 'coverage_end_date', className: 'min-tablet hide-panel', orderable: false}),
-  total_receipts: tables.currencyColumn({data: 'total_receipts', className: 'min-tablet hide-panel'}),
-  total_disbursements: tables.currencyColumn({data: 'total_disbursements', className: 'min-tablet hide-panel'}),
-  total_independent_expenditures: tables.currencyColumn({data: 'total_independent_expenditures', className: 'min-tablet hide-panel'}),
-  modal_trigger: {
-    className: 'all u-no-padding',
-    width: '20px',
+  receipt_date: dateColumn({data: 'receipt_date', className: 'min-tablet'}),
+  coverage_end_date: dateColumn({data: 'coverage_end_date', className: 'min-tablet hide-panel', orderable: false}),
+  total_receipts: currencyColumn({data: 'total_receipts', className: 'min-tablet hide-panel'}),
+  total_disbursements: currencyColumn({data: 'total_disbursements', className: 'min-tablet hide-panel'}),
+  total_independent_expenditures: currencyColumn({data: 'total_independent_expenditures', className: 'min-tablet hide-panel'}),
+  modal_trigger: modalTriggerColumn
+};
+
+var independentExpenditures = [
```

Comment:
This is just an array of objects representing all of the different columns for the independent expenditures, correct?